mhep/mhep/dev/templates/dev/view.html

```diff
@@ -426,18 +435,14 @@ <h3>This scenario is locked!</h3>
         $(window).scrollTop(650);
     });
     $("#openbem").on('click', ".block-header", function () {
+        const project_id = this.closest("[scenario]").getAttribute('scenario');
```

Comment:
When I click 'Create new scenario' now, I'm getting: `TypeError: this.closest(...) is null` in the console.